packages/xchain-thorchain-query/src/thorchain-query.ts

```diff
@@ -283,32 +284,34 @@ export class ThorchainQuery {
     }
     return swapEstimate
   }
-
+  private abbreviateAssetString(asset: Asset): string {
+    const contractAddress = getContractAddressFromAsset(asset)
+    if (contractAddress && contractAddress.length > 5) {
+      const abrev = contractAddress.substring(contractAddress.length - 5)
+      const sep = asset.chain !== Chain.THORChain && asset.synth ? '/' : '.'
+      return `${asset.chain}${sep}${asset.ticker}-${abrev}`
+    }
```

Comment:
(Unit) test it

Reply:
line 251 in this file  has the test  packages/xchain-thorchain-query/__tests__/thorchain-query.test.tspackages/xchain-thorchain-query/__tests__/thorchain-query.test.ts

i also had to modify some  e2e tests to accommodate these changes, so i feel it's tested. do you see something i missed tho?